### DISCUSSION TITLE
docs/intro

### DISCUSSION THREAD

**Discussion by giscus** [gqty-dev/gqty#693] [Q&A]
# docs/intro

a GraphQL client built for rapid iteration.

https://gqty.dev/docs/intro

**Comment by Huzaifa-MS** [comment]
Hi I am using this library in my Nextjs project. I am a beginner so forgive me if what I say doesn't make sense.

How can I use this gql client in the getStaticProps method to fetch some posts from a Wordpress instance?
I am unable to use hooks in that method

**Reply by vicary** [reply]
Read more about SSR here: https://gqty.dev/docs/react/ssr.

**Comment by Jason-zyshen** [comment]
Hi. I really new to GQty and React. I think GQty is great for my project but I couldn't even make a simple useQuery work. Could you just provide a very basic workable code to illustrate how to use GQty to connect to a GraphQL endpoint and fetch data. Thanks in advance.

**Reply by vicary** [reply]
I see that you already have some success at #1142, feel free to ask if you have more questions.
